src/conduit/components/header.cljs

```diff
@@ -32,8 +32,13 @@
     (when icon " ")
     label]])
 
-(rum/defc Header [r route {:keys [loading? logged-in?]}]
-  (let [user-nav-items (filter #(not= (if logged-in? :non-logged :logged) (:display-for %)) nav-items)]
+(rum/defc Header [r route {:keys [loading? current-user]}]
+  (let [user-nav-items (->> nav-items
+                            (filter #(not= (if current-user :non-logged :logged) (:display-for %)))
+                            (#(if current-user
```

Comment:
the whole threading macro could be more readable here, a quick refactoring would be nice